src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -244,7 +320,60 @@ describe('ReviewScreen', () => {
         'fiatConnectReviewScreen.paymentMethodVia, {"providerName":"Provider Two"}',
       ])
     })
-
+    it('disables the submit button if feeInfo is not available', () => {
```

Comment:
i guess if this is what we do for the Send flow it's parity and worth merging for now, but this seems like kind of a confusing UX. To just have the submit button greyed out with no reason why.

Reply:
maybe we should log when this happens so we can figure out how frequent it is and prioritize possible improvements

Reply:
We have some existing events that should cover that well. I'll put together a report